src/reducers/targetReducer.js

```diff
@@ -0,0 +1,42 @@
+import * as types from '../actions/actionTypes';
+import mapInitialState from './mapInitialState';
+
+const targetReducer = (state = mapInitialState, action) => {
+  switch (action.type) {
+    case types.START_NEW_TARGET: {
+      return state.set('addingNewTarget', true)
+        .set('newTargetLat', action.payload.lat)
+        .set('newTargetLong', action.payload.lng);
+    }
+    case types.END_NEW_TARGET: {
+      return state.set('addingNewTarget', false)
+        .set('targetRadius', mapInitialState.get('targetRadius'))
+        .set('targetErrors', {});
+    }
+    case types.UPDATE_RADIUS: {
+      return state.set('targetRadius', action.payload);
+    }
+    case types.CREATE_TARGET: {
+      return state.set('targetRadius', action.payload);
```

Comment:
This one and the previous action do the same. Can you rename it to something more generic and reuse it?